src/Model/Release/Version.php

```diff
@@ -26,17 +26,17 @@ class Version
     /**
      * @var int
```

Comment:
get rid of the docblock types if you're going to properly define them

Reply:
Done